weave/trace/op.py

```diff
@@ -89,6 +89,7 @@ async def anext(obj: AsyncIterator[V], default: Optional[V] = None) -> V:  # noq
 CALL_CREATE_MSG = "Error creating call:\n{}"
 ASYNC_CALL_CREATE_MSG = "Error creating async call:\n{}"
 ON_OUTPUT_MSG = "Error capturing call output:\n{}"
+UNINITIALIZED_MSG = "Warning: Traces will not be logged. Call weave.init to log your traces to a project.\n"
```

Comment:
nit: remove `Warning: `

it will be included by default when you do `logger.warn`